src/Employee.java

```diff
@@ -0,0 +1,92 @@
+import java.util.Objects;
+
+public class Employee {
+    private String FullName;
+    private double salary;
+    private int otdel;
+    private int id;
+    private static int idCount = 1; // для подсчета ID
+
+    public Employee (String FullName, int otdel, double salary) {
```

Comment:
```suggestion
    public Employee (String fullName, int otdel, double salary) {
```
названия параметров надо писать с маленькой буквы